src/WrapOfferer.sol

```diff
@@ -1,51 +1,54 @@
 // SPDX-License-Identifier: CC0-1.0
-pragma solidity ^0.8.20;
+pragma solidity 0.8.20;
 
 import {ContractOffererInterface, IWrapOfferer, ReceiptFillerType, ExpiryType} from "./interfaces/IWrapOfferer.sol";
 
 import {ReceivedItem, SpentItem, Schema} from "seaport-types/src/interfaces/ContractOffererInterface.sol";
 import {ItemType} from "seaport-types/src/lib/ConsiderationEnums.sol";
 
-import {IDelegateToken, TokenType} from "./interfaces/IDelegateToken.sol";
+import {IDelegateToken, IDelegateRegistry} from "./interfaces/IDelegateToken.sol";
 
-import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
-import {IERC721} from "openzeppelin-contracts/contracts/token/ERC721/IERC721.sol";
-import {IERC1155} from "openzeppelin-contracts/contracts/token/ERC1155/IERC1155.sol";
+import {IERC20} from "openzeppelin/token/ERC20/IERC20.sol";
+import {IERC721} from "openzeppelin/token/ERC721/IERC721.sol";
+import {IERC1155} from "openzeppelin/token/ERC1155/IERC1155.sol";
 
 /// @notice A Seaport ContractOfferer
 contract WrapOfferer is IWrapOfferer {
     uint256 internal constant CONTEXT_SIZE = 59;
 
     /// @notice Address for Seaport 1.5
-    address public immutable SEAPORT;
+    address public immutable seaport;
```

Comment:
Isn't it best practice to have constants/immutables be uppercase to distinguish from mutable vars?

Reply:
Immutables should be lowercase according to the solidity style guide